cmd/minikube/cmd/start.go

```diff
@@ -202,6 +202,16 @@ func runStart(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if existing != nil && existing.KubernetesConfig.ContainerRuntime == "crio" && driver.IsKIC(existing.Driver) {
```

Comment:
I'm not following this -- doesn't this run only if the previous provisionWithDriver error == nil?

Reply:
correct, provision isn't what fails for crio, it's starting k8s that fails. what I found works is if we just stop and start again here, everything works as expected.

I'm basically emulating restarting and failing, then calling minikube stop then calling minikube start again, except I'm skipping the 6 minutes we need to wait for apiserver to not come back in the startWithDriver portion of the code.

again, I have no idea why, but empirically it works and it makes the tests pass.